config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
Eh, the golang kernel is a bit of a hack, I don't think I'd ever use it in a notebook. Compiled software really isn't suited for notebook environments.

True about R devs, but most R devs are happy rstudio users and would likely opt to use that first.

Good points, I rescind my complaints about a short name if a longer name is conveying more useful information.